### ISSUE TITLE
Running Apollo Codegen in my project folder try to generate from apollo-ios package .graphql files

### ISSUE THREAD

**Issue by Dimillian** [apollographql/apollo-ios#2602]
Hello,

Trying to upgrade our project from 0.5.X to 1.0.2, updated our Swift codegen package successfully (well at least to compile) and i'm running into new errors. 

You can see here the error when running `ApolloCodegen.build(with: configuration, withRootURL: hangtagFolderURL)`

```
/Users/dimillian/Documents/Developer/hangtag/Codegen/.build/checkouts/apollo-ios/Tests/TestCodeGenConfigurations/Other-CustomTarget/GraphQLAPI/graphql/ccnGraphql/ClassroomPetsCCN.graphql:2:16
1 | query ClassroomPetsCCN {
2 |   classroomPets[!]? {
  |                ^
3 |     ...ClassroomPetDetailsCCN
```

I have a very basic configuration

```
let configuration = ApolloCodegenConfiguration(schemaName: "GQL",
                                               input: .init(schemaPath: urlToSchemaFile.absoluteString),
                                               output: .init(schemaTypes: .init(path: outputURL.absoluteString,
                                                                                moduleType: .swiftPackageManager)))
```

How do I exclude Xcode SPM checkout / build folders? It was not trying to generate from these folders with the javascript CLI, is it a bug with the new Swift based codegen? 

Thanks!

**Comment by Dimillian** [comment]
If I'm specifying directories where I want to search for operations:

`operationSearchPaths: ["hangtag/**/*.graphql", "Packages/**/*.graphql"])`

I get another error:

```
[DEBUG - ApolloCodegenLib:Glob.swift:194] - Matching /Users/dimillian/Documents/Developer/Medium/hangtag/Packages/Model/.swiftpm/xcode/xcshareddata/*.graphql
[DEBUG - ApolloCodegenLib:Glob.swift:194] - Matching /Users/dimillian/Documents/Developer/Medium/hangtag/Packages/Model/.swiftpm/xcode/xcshareddata/xcschemes/*.graphql
ApolloCodegenLib/JavaScriptBridge.swift:343: Precondition failed: Expected JavaScript array but found: undefined

```

**Comment by calvincestari** [comment]
Hi @Dimillian 👋🏻 - yes, this is a bug in the new Swift-based Codegen. We're got https://github.com/apollographql/apollo-ios/issues/2552 to track a fix for it and it will be included in the next release, `1.0.3`.

As for the second error, I've only seen that when the schema file cannot be located. Does the log output show that it finds your schema file?

**Comment by Dimillian** [comment]
Thanks!

**Comment by calvincestari** [comment]
I'm going to close this issue and we can track the progress of the bug in #2552.

**Comment by calvincestari** [comment]
@Dimillian, this is fixed with the merging of #2628 and will be included with `1.0.3`. There is maybe one or two other issues I need to check on first before cutting that next release.

**Comment by Dimillian** [comment]
@calvincestari Thanks, all good for this issue, which led me to another issue https://github.com/apollographql/apollo-ios/issues/2645 :)
